### ISSUE TITLE
Keyboard focus methods are missing

### ISSUE THREAD

**Issue by hernanmd** [pharo-spec/Spec#1668]
How to reproduce:

```smalltalk
| textPresenter |

textPresenter := SpTextInputFieldPresenter new.
textPresenter 
	open;
	hasKeyboardFocus.
```

And

```smalltalk
| textPresenter |

textPresenter := SpTextInputFieldPresenter new.
textPresenter 
	open;
	focusedPresenter.
```
